dso/model.py

```diff
@@ -283,38 +283,17 @@ def key(self) -> str:
 
 
 @dataclasses.dataclass(frozen=True)
-class MalwareFindingBase():
+class ClamAVMalwareFinding(Finding):
+    layer_digest: str | None # oci-image specific
```

Comment:
wo do not always have a _layer_-digest, do we? how about we either omit it (might be included in filename), or make it more generic (e.g. artefact_digest, or similar)

Reply:
the layer-digest is specific to oci-artefacts
e.g. gardenlinux stores (some) of its artefacts in s3 buckets without oci manifests

Reply:
> we do not always have a layer-digest, do we
We do not always have a layer-digest.

Reply:
We could make it more generic to allow re-use, I don't have a strong opinion here.

Reply:
in that case, I suggest to rename it (ideally also fill value for s3-artefacts). do you happen to know for which purpose we actually keep layer-digest? as mentioned before, I think we could include layer-digest in path (filename), which should be "good enough"; I do not recall storing layer-digest to yield an additional benefit 🤔

Reply:
> do you happen to know for which purpose we actually keep layer-digest?

I think we never really discussed this in detail, but I think this is required due to overwrite semantics of multiple oci layers for the very same filepath